oak-jcr/src/main/java/org/apache/jackrabbit/oak/jcr/session/NodeImpl.java

```diff
@@ -1679,6 +1679,24 @@ public PropertyImpl performNullable() throws RepositoryException {
         });
     }
 
+    @Override
+    public Node getParentOrNull() throws RepositoryException {
```

Comment:
i would annotate the method's return value with @Nullable to make it super clear.

Reply:
done